js/view.js

```diff
@@ -6,7 +6,30 @@
     $('.main-nav').on('click', '.tab', function(){
       $('section').hide();
       $('#' + $(this).data('content')).fadeIn();
-      console.log($(this).data('content'));
+    });
+  };
+
+  projectView.populateFilter = function() {
+    $('article').each(function() {
```

Comment:
From what I can tell, $('article') isn't returning anything and I'm not sure why. $('section') works, and I can't see a reason why $('article') isn't. You could try adding a distinct class to your article template and work around it like that.